code/index.html

```diff
@@ -1,35 +1,71 @@
-  </head>
-  <body>
-    <section class="cards">
-      <div class="killer-deck" id="killer" onclick="pickKiller()">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:600,900&display=swap" rel="stylesheet">
+  <title>Clue</title>
+</head>
+
+<body>
+  <header class="game-header">
+    <h1 class="game-title">Cluedo</h1>
+  </header>
+  <section class="cards">
+    <div class="deck" id="killer" onclick="pickKiller()">
+      <div class="card-inner">
+        <div class="card-front">
+          <p class="card-type">Killer</p>
+        </div>
+        <div class="card-back">
+          <img id="killerImage" class="killer-portrait" src="" />
+          <p id="killerColor" class="killer-details">
+            <span id="killerName" class="killer-name"></span>, <span id="killerAge" class="killer-age"></span><br />
+          </p>
+          <p id="killerOccupation" class="badge"></p>
+          <p id="killerDescription" class="killer-description"></p>
+          <p class="card-type">Killer</p>
+        </div>
       </div>
-      <div class="weapon-deck" id="weapon" onclick="pickWeapon()">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
+    </div>
+    <div class="deck" id="weapon" onclick="pickWeapon()">
+      <div class="card-inner">
+        <div class="card-front">
+          <p class="card-type">Weapon</p>
+        </div>
+        <div class="card-back">
+          <img id="weaponImage" class="weapon-image" src="" />
+          <p id="weaponName" class="weapon-name"></p>
+          <p id="weaponWeight" class="badge"></p>
+          <p id="weaponDescription" class="weapon-description"></p>
+          <p class="card-type">Weapon</p>
+        </div>
       </div>
-      <div class="room-deck" id="room" onclick="pickRoom()">
-        <p>ROOM</p>
-        <p id="roomName"></p>
+    </div>
+    <div class="deck" id="room" onclick="pickRoom()">
+      <div class="card-inner">
+        <div class="card-front">
+          <p class="card-type">Room</p>
+        </div>
+        <div class="card-back">
+          <img id="roomImage" class="room-image" src="" />
+          <p id="roomName" class="room-name"></p>
+          <p id="roomDescription" class="room-description"></p>
+          <p class="card-type">Room</p>
+        </div>
       </div>
-    </section>
-    <h1 id="mystery"></h1>
-    <button onclick="revealMystery()">REVEAL THE CRIME</button>
+    </div>
+    </div>
+  </section>
+  <section class="mystery">
+    <h1 id="mystery" class="mystery-title"></h1>
```

Comment:
This looks like it will suit better as a `<h2>` element because you already have a `<h1>` that describes the page